samples/algorithms/QuantumHelloWorld.qs

```diff
@@ -0,0 +1,24 @@
+/// # Quantum Hello World!
+///
+/// This Q# code generates a random bit by setting a qubit in a superposition of the
+/// computational basis states |0〉 and |1〉, and returning the measurement result.
+namespace QuantumHelloWorld {
+    // Import the `Message` function and the `H` operation used in the program.
+    open Microsoft.Quantum.Intrinsic;
+
+    @EntryPoint()
+    operation RandomBit() : Result {
+        Message("Hello world!");
+        use qubit = Qubit();
```

Comment:
A comment about qubit allocation and automatic scoping could be helpful here.

Reply:
Added a brief comment on qubit allocation and scoping.

Reply:
There will be a "Random Bit" specific sample later so I'll include this comment there.